### PR TITLE
Add the Kaldheim Commander set

### PR DESCRIPTION
**Pull request by doggymien4o**
Would like these added: Wolverine Riders and Lathril, Blade of the Elves. They make green 1/1 Elf Warrior tokens

### CODE REVIEW

File: tokens.xml [hunk 1]
```diff
@@ -2564,6 +2564,7 @@ When Earthshaker Khenra enters the battlefield, target creature with power less
             <reverse-related count="2">Gilt-Leaf Ambush</reverse-related>
             <reverse-related count="3">Hunting Triad</reverse-related>
             <reverse-related>Imperious Perfect</reverse-related>
+            <reverse-related>Lathril, Blade of the Elves</reverse-related>
```
Comment (ebbit1q) [inline tokens.xml:2567]:
```suggestion
            <reverse-related count="x=2">Lathril, Blade of the Elves</reverse-related>
```

You can add an amount of tokens to be created like this :)

**Comment by tooomm** [comment]
Kaldheim Commander (KHC) isn't added at all yet.

There are other tokens and creators which are also not linked:
https://scryfall.com/sets/tkhc
https://scryfall.com/search?q=set%3Akhc+o%3Atoken

**Comment by ebbit1q** [comment]
> ~Kaldheim Commander (KHC) isn't added at all yet.~
> 
> ~There are other tokens and creators which are also not included as of now:~
> ~https://scryfall.com/sets/tkhc~
> ~https://scryfall.com/search?q=set%3Akhc+o%3Atoken~
> 
> Scrape that. Psithief included them this time in the normal KHM commit.

those are mostly reprints, please use this search
https://scryfall.com/search?q=set%3Akhc+o%3Atoken+-is%3Areprint

**Comment by tooomm** [comment]
Spotted that as well. Updated the comment in the meantime.

**Comment by tooomm** [comment]
@doggymien4o Thanks for bringing this up to our attention and even applying the needed changes in this PR! 👍 

I added Ranar as well, KHC is probably complete then.

**Comment by ebbit1q** [comment]
> @doggymien4o Thanks for bringing this up to our attention and even applying the needed changes in this PR! +1
> 
> I added Ranar as well, KHC is probably complete then.

please sort it alphabetically

**Review by ebbit1q — APPROVED** [review]
:+1:

**Comment by Psithief** [comment]
I will check, rebase and squash this.

**Comment by Psithief** [comment]
Because this branch had multiple authors, I've squashed the commits by author before merging.
See 69687a4bf68ba71effd1923b56a5371b0b4dfcbd, 4bd35e7bb538307adef1af0b9b2fa7c91f43c4a8 and b862721a0cdcd03a32658213f32178800221c6d0.

**Comment by Psithief** [comment]
I just wanted to see what happens in git if I do this merge manually.

**Comment by tooomm** [comment]
@doggymien4o This change is live now since yesterday. You have to rerun the card updater `Help --> Check for card updates` in order to get the new tokens with your fix included! Thanks again 👍

**Pull request closed without merging**